common/src/main/scala/tsec/common/ManagedRandom.scala

```diff
@@ -32,6 +32,12 @@ trait ManagedRandom {
     cachedRand = tmpRand
   }
 
+  private[tsec] def forceIncrement: Unit = {
```

Comment:
This mutable behavior is private because most of the time, people should be using `nextBytes`.

calling forcedIncrement is only for when we pass the mutable reference around, with care.